thirdparty/github.go

```diff
@@ -727,7 +727,7 @@ func doGithubRequest(client *http.Client, req *http.Request, accept string) (str
 
 // buildPatchURL creates a URL to enable downloading patch files through the
 // Github API
-func buildPatchURL(gp *patch.GithubPatch) string {
+func buildPatchURL(gp patch.GithubPatch) string {
```

Comment:
This was expecting a pointer for some reason.
Many of the changes in the PR are just to pass `gp` by value.